handbook/company/communications.md

```diff
@@ -565,7 +565,15 @@ The images used in the docs live in `docs/images/`. Note that you must provide t
 
 > Note that the instructions above also apply to adding images in the Fleet handbook.
 
+### Audit logs
 
+The [Audit logs doc page](../../docs/Using%20Fleet/Audit-logs.md) has a page generator that is used to speed up doc writing when Fleet adds new activity types.
+
+1. If you're making a copy change to an exiting activity type or adding a new activity type, edit the `activities.go` file [here](https://github.com/fleetdm/fleet/blob/main/server/fleet/activities.go).
```

Comment:
Nit: Add a reminder that when adding a new activity type, it must be added to the `ActivityDetailsList`.

Reply:
✅